functions/go/fix/main.go

```diff
@@ -36,10 +36,10 @@ func (fp *FixProcessor) Process(resourceList *framework.ResourceList) error {
 	resourceList.Result = &framework.Result{
 		Name: "fix",
 	}
-	resourceList.Items, err = s.Filter(resourceList.Items)
+	_, err = s.Filter(resourceList.Items)
```

Comment:
Why? Does this `Filter` modify the items in-place?

Reply:
We do modify resources in-place but if I think more about it, it is better to leave it as is in general. Reverted.